code/datums/materials/_material.dm

```diff
@@ -24,6 +24,8 @@ Simple datum which is instanced once per type and is used for every object of sa
 	var/strength_modifier = 1
 	///This is a modifier for integrity, and resembles the strength of the material
 	var/integrity_modifier = 1
+	///Armor modifiers, multiplies an items normal armor vars by these amounts.
+	var/armor_modifiers = list("melee" = 1, "bullet" = 1, "laser" = 1, "energy" = 1, "bomb" = 1, "bio" = 1, "rad" = 1, "fire" = 1, "acid" = 1)
```

Comment:
gonna be impossible to balance

Reply:
I don't think you should be able to build any armor out of any material, this is more for edge-cases such as this one, or things that we can balance around this due to them only using this system, and thus having easily adjustable base numbers (walls, windows, doors)

Reply:
Yeah as far as I can tell this isn't applicable to any armors except the one being added in this PR, balance shouldn't be a concern